lte/gateway/c/oai/lib/hashtable/hashtable.c

```diff
@@ -374,6 +374,9 @@ hashtable_rc_t hashtable_ts_destroy(hash_table_ts_t* hashtblP) {
   free_wrapper((void**) &hashtblP->nodes);
   bdestroy_wrapper(&hashtblP->name);
   free_wrapper((void**) &hashtblP->lock_nodes);
+  if (hashtblP->lock_attr) {
```

Comment:
No need for a null check, free_wrapper(&hashtblP->lock_attr) will call free(hashtblP->lock_attr), which does nothing if `hashtblP->lock_attr` is NULL

Reply:
In free_wrapper(), 
void free_wrapper(void** ptr) {
 44   // for debug only
 45   AssertFatal(ptr, "Trying to free NULL ptr");
 46   if (ptr) {
 47     free(*ptr);
 48     *ptr = NULL;
 49   }
 50 }

At line number 46, I think we need to check if(*ptr) as well.
Which would look something like this if (ptr && *ptr)

Reply:
From https://man7.org/linux/man-pages/man3/free.3.html
```
The free() function frees the memory space pointed to by ptr, which
must have been returned by a previous call to malloc(), calloc(), or
realloc().  Otherwise, or if free(ptr) has already been called
before, undefined behavior occurs.  If ptr is NULL, no operation is
performed.
```
So we don't need to check if the argument to free (in our case `*ptr`) is Null.